keyboards/kprepublic/bm43hsrgb/rules.mk

```
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```

This will save me a *tiny* bit of work later - `BOOTMAGIC_ENABLE = lite` is going away in a few months. This setting is equivalent.

```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```